lib/FroalaEditorFunctionality.jsx

```diff
@@ -48,6 +48,7 @@ export default class FroalaEditorFunctionality extends React.Component {
   }
 
   componentWillUnmount() {
+    this.element=null
```

Comment:
Why is this required? This will cause the `destroyEditor` method to fail.

Reply:
Hi @harasunu-narayan 

I have reverted back this line of code as it was causing `destroyEditor` method to fail